e2e/save_report.js

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/* eslint-disable no-console */
+
+/*
+ * This is used for saving artifacts to AWS S3, sending data to automation dashboard and
+ * publishing quick summary to community channels.
+ *
+ * Usage: [ENV] node save_report.js
+ *
+ * Environment variables:
+ *   BRANCH=[branch]            : Branch identifier from CI
+ *   BUILD_ID=[build_id]        : Build identifier from CI
+ *   BUILD_TAG=[build_tag]      : Docker image used to run the test
+ *
+ *   For saving artifacts to AWS S3
+ *      - AWS_S3_BUCKET, AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
+ *   For saving test cases to Test Management
+ *      - TM4J_ENABLE=true|false
+ *      - TM4J_API_KEY=[api_key]
+ *      - JIRA_PROJECT_KEY=[project_key], e.g. "MM",
+ *      - TM4J_FOLDER_ID=[folder_id], e.g. 847997
+ *   For sending hooks to Mattermost channels
+ *      - FULL_REPORT, WEBHOOK_URL and DIAGNOSTIC_WEBHOOK_URL
+ *   Test type
+ *      - TYPE=[type], e.g. "MASTER", "PR", "RELEASE", "CLOUD"
+ */
+
+const path = require('path');
+
+const chai = require('chai');
+const generator = require('mochawesome-report-generator');
+
+const {
+    //generateDiagnosticReport,
+    generateShortSummary,
+    generateTestReport,
+    removeOldGeneratedReports,
+    sendReport,
+    readJsonFromFile,
+    writeJsonToFile,
+} = require('./utils/report');
+const {saveArtifacts} = require('./utils/artifacts');
+const {MOCHAWESOME_REPORT_DIR} = require('./utils/constants');
+const {createTestCycle, createTestExecutions} = require('./utils/test_cases');
+
+require('dotenv').config();
+
+const saveReport = async () => {
+    const {
+        BRANCH,
+        BUILD_ID,
+        BUILD_TAG,
+        // DIAGNOSTIC_WEBHOOK_URL,
+        // DIAGNOSTIC_USER_ID,
+        // DIAGNOSTIC_TEAM_ID,
+        FAILURE_MESSAGE,
+        TM4J_ENABLE,
+        TM4J_CYCLE_KEY,
```

Comment:
Please change `TM4J` to `ZEPHYR` being the new name. We'll update the ones in mattermost-webapp, but [other repo started using Zephyr already](https://github.com/mattermost/mattermost-mobile/pull/6560/files#diff-4feac1abcee84ad8446fc56c50ddebd56a090571d90a26718c0f917291052c2fR56-R57).
```suggestion
        ZEPHYR_ENABLE,
        ZEPHYR_CYCLE_KEY,
```